extensions/data-workspace/src/dialogs/projectDashboard.ts

```diff
@@ -110,4 +106,86 @@ export class ProjectDashboard {
 
 		return button;
 	}
+
+	private createContainer(title: string, location: string): azdata.FlexContainer {
+		const dashboardData: IDashboardTable[] = this.projectProvider!.dashboardComponents;
+
+		const rootContainer = this.modelView!.modelBuilder.flexContainer().withLayout(
+			{
+				flexFlow: 'column',
+				width: '100%',
+				height: '100%'
+			}).component();
+
+		const titleLabel = this.modelView!.modelBuilder.text()
+			.withProperties<azdata.TextComponentProperties>({ value: title, CSSStyles: { 'margin-block-start': '20px', 'margin-block-end': '0px' } })
+			.component();
+		rootContainer.addItem(titleLabel, { CSSStyles: { 'padding-left': '34px', 'padding-top': '15px', 'font-size': '36px', 'font-weight': '400' } });
+
+		const projectFolderPath = vscode.Uri.file(path.dirname(location)).fsPath;
```

Comment:
can this just be path.dirname(location)?

Reply:
Yes. Updated it appropriately.